adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationActivityUnitTest.java

```diff
@@ -87,6 +87,8 @@
 
     private static final int TEST_CALLING_UID = 333;
 
+    private static final String TEST_AUTHORITY = "https://login.microsoftonline.com/common";
```

Comment:
What will happen if keeping using the pseudo authority url here? Will the unit test fail?

Reply:
it will pass the UT. Since there's some code change in checking the authority, so we need to provide valid authority in UT.